guillotina/contrib/dbusers/content/users.py

```diff
@@ -85,6 +88,15 @@ def groups(self):
     def _properties(self):
         return {}
 
+    async def set_password(self, new_password, old_password=None):
+        if old_password is not None:
+            valid = check_password(self.password, old_password)
```

Comment:
Do we need to add something to define some password constraints, like:
minium length, Caps chars, lower cars, numbers... (The basic one could be just a minium length).

Reply:
I don't think that is the scope of this PR. Right now can be guided on the frontend and not enforced on the backend.

Reply:
I've added a validation schema for validation process to reset password can be validated and minlength can be set